compiler/backend/presLangScript.sml

```diff
@@ -46,8 +51,7 @@ val _ = Datatype`
     | Lit tra lit
       (* Constructor application.
        A Nothing constructor indicates a tuple pattern. *)
```

Comment:
"A nothing constructor indicates a tuple pattern", seems a bit off since you can no longer directly see that conF can be set to none. We should probably, connect this comment clearly to conF at line 22, not sure as to how though.